source/brailleDisplayDrivers/handyTech.py

```diff
@@ -535,6 +553,7 @@ def getManualPorts(cls):
 
 	def __init__(self, port="auto"):
 		super(BrailleDisplayDriver, self).__init__()
+		wx.CallAfter(self.create_message_window)

```

Comment:
Could you add a comment saying why this is wrapped in a CallAfter, i.e. because we need to make sure this is performed on the main thread?